msteams-platform/tabs/how-to/authentication/tab-sso-overview.md

```diff
@@ -0,0 +1,203 @@
+
+### Use cases for enabling SSO for tab
+
+Here are some use cases where enabling SSO is beneficial. Call `getAuthToken` in these scenarios to use Teams identity for obtaining access token for your app users:
+
+1. To get user’s teams identity from Teams if you have an existing app that you want to be available within a tab app in Teams for your app users.
+
+2. To authenticate a user by reusing the team’s identity inside your Tab application.
+
+3. To authenticate and get user’s teams identity without having them to signing in again inside configurable tabs. This is applicable to some settings need to be configured at an config stage.
+
+4. To obtain an access token inside a task module, when it's invoked from a tab, bots, messaging extension, or adaptive cards.
+
+5. To authenticate a user in stage view.
+
+The SSO API also works in [task modules](../../../task-modules-and-cards/what-are-task-modules.md) that embed web content.
+
+> [!NOTE]
+> Tabs are Teams-aware web pages. To enable SSO in a web page hosted inside a tab, add [Teams Javascript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), and call `microsoftTeams.initialize()`. Once you've initialized Microsoft Teams, call `microsoftTeams.getAuthToken()`, which will get the access token for your app.
+
+To achieve Teams SSO, you must configure your app to enable SSO for authenticating and authorizing your users.
+
+## Enable SSO for a tab app
+
+This section describes the tasks involved in implementing SSO for a tab app. These tasks are language- and framework-agnostic.
+
+To build a tab app that uses Teams SSO to authenticate users:
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; :::image type="content" source="../../../assets/images/authentication/teams-sso-tabs/teams-sso-accesstoken.png" alt-text="Steps to enable SSO for tab" border="false":::
+
+1. Register with Azure AD: Create an Azure AD app to generate an app ID and app ID URI You also configure redirect URI where Azure AD would send the access token in exchange for identity token for the current user logged into Teams.
+2. Configure code: Update the Teams client to handle access token, calling it when a user accesses your app and validating it when received.
+3. Teams manifest: Update your Teams client manifest with the app ID generated on Azure AD and the app ID URI.
+
+## Best practices
+
+Here's a list of best practices:
+
+- **Don’t cache the access token**: Never cache or store the access token in your app's client-side code. Always call `getAuthToken` when you need an access token. Teams Client caches the access token (or request a new one if it expires). This makes sure that there's no accidental leak of your token from your web-app.
+
+## Known limitations
+
+- SSO in teams only work with OAuth 2.0 protocol.  
+- SSO only works with Azure AD. To extend it to different OAuth Identity providers, the flow needs to be implemented.
```

Comment:
> the flow needs to be implemented

It's not clear which flow we are referring here.  This should ideally be linked to some other part of the doc where we talk about solution.

Reply:
Added links to oAuth flow in Tabs  and Azure AD authentication pages.